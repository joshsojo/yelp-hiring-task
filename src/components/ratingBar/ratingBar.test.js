import { shallow } from "enzyme";
import React from "react";
import RatingBar from "./ratingBar.component";
describe("Rating Star Bar Component", () => {
  test("Expecting component to return Nothing when passed no props", () => {
    expect(shallow(<RatingBar />)).toEqual({});
  });
  test("Expecting component to return no value with props passed", () => {
    expect(shallow(<RatingBar rating={4.5} />)).toEqual({});
  });
});
