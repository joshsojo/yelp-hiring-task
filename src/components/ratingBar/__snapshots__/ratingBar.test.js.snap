// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rating Star Bar Component Renders without crashing 1`] = `ShallowWrapper {}`;
